devices/YarpRobotLoggerDevice/app/robots/iCubGenova09/yarp-robot-logger.xml

```diff
@@ -61,12 +61,14 @@ GNU Lesser General Public License v2.1 or any later version. -->
       <elem name="left_rear_ft_client">left_rear_ft_client</elem>
       <elem name="right_front_ft_client">right_front_ft_client</elem>
       <elem name="right_rear_ft_client">right_rear_ft_client</elem>
+      <elem name="right_upper_leg_ft_client">right_upper_leg_ft_client</elem>
       <elem name="left_front_wrench_client">left_front_wrench_client</elem>
       <elem name="left_rear_wrench_client">left_rear_wrench_client</elem>
       <elem name="right_front_wrench_client">right_front_wrench_client</elem>
       <elem name="right_rear_wrench_client">right_rear_wrench_client</elem>
       <elem name="right_arm_wrench_client">right_arm_wrench_client</elem>
       <elem name="left_arm_wrench_client">left_arm_wrench_client</elem>
+      <elem name="right_upper_leg_wrench">right_upper_leg_wrench</elem>
```

Comment:
why don't we use the same naming convention as the others? 
```suggestion
      <elem name="right_upper_leg_wrench">right_upper_leg_wrench_client</elem>
```

Reply:
Sorry @prashanthr05 my fault, I forgot to write client. I pushed the modifications to be aligned with the others.